controllers/multiclusterobservability/multiclusterobservability_controller.go

```diff
@@ -103,6 +103,14 @@ func (r *MultiClusterObservabilityReconciler) Reconcile(ctx context.Context, req
 		return ctrl.Result{}, err
 	} else if isTerminating {
 		reqLogger.Info("MCO instance is in Terminating status, skip the reconcile")
+		svmCrdExists, err := util.CheckCRDExist(r.CrdClient, config.StorageVersionMigrationCrdName)
```

Comment:
Can we add ownerreference for this CR so that we do not need to special handle the deletion case?

Reply:
Actually I tried that, I got error of `can't add owner ownerreference to cluster-scope resource` because `StorageVersionMigration` is cluster-scoped resource.
It's weird because MCO CR is also cluster scoped resource, it should be working.

Now I have to add logic to delete the `StorageVersionMigration` resource after MCO CR is deleted, we can't depend on the kubernetes GC to delete it because we can't add ownerreference.